### PR TITLE
repositories: Add dinolay

### PR DESCRIPTION
**Pull request by TruncatedDinosour**
[This](https://github.com/TruncatedDinosour/dinolay) overlay consists of:
- Packages I use, but they're not in the gentoo main repo
- Utilities I made and made them into an ebuild
- Custom ebuilds
- Git packages
- Packages that are in the gentoo repos, but they don't meet my satisfaction (like transmission, but I want transmission-cli)

### CODE REVIEW

**Review by jsmolic — CHANGES_REQUESTED** [review]
Please keep the list sorted

**Comment by TruncatedDinosour** [comment]
> Please keep the list sorted

What list? How sorted?

**Comment by TruncatedDinosour** [comment]
> Please keep the list sorted

okay

**Comment by TruncatedDinosour** [comment]
> Please keep the list sorted

done

**Comment by TruncatedDinosour** [comment]
> Please keep the list sorted

done

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -1025,6 +1025,19 @@
     <feed>https://cgit.gentoo.org/dev/dilfridge.git/atom/</feed>
     <!-- <feed>https://cgit.gentoo.org/dev/dilfridge.git/rss/</feed> -->
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>dinolay</name>
+    <description lang="en">My gentoo overlay for me to not need to use multiple ones</description>
```
Comment (jsmolic) [inline files/overlays/repositories.xml:1030]: This should be changed to "dinolay's overlay" or something similar

Comment (TruncatedDinosour) [reply files/overlays/repositories.xml:1030]: Sure

Comment (TruncatedDinosour) [reply files/overlays/repositories.xml:1030]: wait, which part, the description or the name?

**Comment by TruncatedDinosour** [comment]
@jsmolic fixed

**Review by jsmolic — APPROVED** [review]
Thanks, I'm going to squash and merge

**Pull request closed without merging**